EMIP/magicscrolls/EMIPms00470/EMIPms00470.xml

```diff
@@ -89,6 +89,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change when="2022-09-02" who="CS">added include statements to incorporate transkribus output</change>
          <change when="2022-09-23" who="RL">Corrected include code errors</change>
          <change when="2022-09-26" who="RL">Removed text body code lines</change></revisionDesc>
+         <change when="2024-08-27" who="RL">Corrected formatting of text file</change>
```

Comment:
sorry this element must go inside of revisionDesc

Reply:
It should be
```
<revisionDesc>
  <change>...</change>
  <change>...</change>
  <change>...</change>
</revisionDesc>
```

Reply:
Now changed - sorry for a simple error here!